Web/Views/BallotIssues/Edit.cshtml

```diff
@@ -22,12 +30,12 @@
             <input type="hidden" asp-for="BallotIssueId" />
             <div class="form-group">
                 @Html.LabelFor(ballotIssue => ballotIssue.BallotIssueTitle, new { @class = "control-label" })
-                @Html.TextAreaFor(ballotIssue => ballotIssue.BallotIssueTitle, new { @class = "form-control", @rows = 2 })
+                @Html.TextAreaFor(ballotIssue => ballotIssue.BallotIssueTitle, new { @class = "form-control", @rows = 2})
                 @Html.ValidationMessageFor(ballotIssue => ballotIssue.BallotIssueTitle, "", new { @class = "text-danger" })
             </div>
             <div class="form-group">
                 @Html.LabelFor(ballotIssue => ballotIssue.Description, new { @class = "control-label" })
-                @Html.TextAreaFor(ballotIssue => ballotIssue.Description, new { @class = "form-control", @rows = 4 })
```

Comment:
is this formatting change necessary?

Reply:
Not necessary, might've been to fix a typo